.github/workflows/release.yml

```diff
@@ -40,6 +56,68 @@ jobs:
       run: |
         python setup.py bdist_wheel sdist
 
+    - name: Save package
+      if: matrix.python == '3.7'
```

Comment:
Can we upload a version for both, since we're already here?

Reply:
Well we only end up pushing one build to PyPI (arbitrarily the one built in 3.7), but that distribution has to work for both 3.6 and 3.7. So I figured I would structure the CI in the same way: build the package in one version of Python but test it across all versions.